src/main/java/ch/njol/skript/entity/BeeData.java

```diff
@@ -0,0 +1,106 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * Copyright 2011-2017 Peter Güttinger and contributors
+ */
+package ch.njol.skript.entity;
+
+import java.util.Random;
+
+import org.bukkit.entity.Bee;
+import org.eclipse.jdt.annotation.Nullable;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.lang.Literal;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+
+public class BeeData extends EntityData<Bee> {
+	
+	static {
+		if (Skript.classExists("org.bukkit.entity.Bee")) {
+			EntityData.register(BeeData.class, "bee", Bee.class, 2,
+				"no nectar bee", "happy bee", "bee", "nectar bee", "angry bee", "angry nectar bee");
```

Comment:
The bee's anger is a time value, we should have an effect to change it accordingly instead of an entity type.

Reply:
Yeah, I figured at some point we could do an effect/expression for it, but I also thought it would be great to quickly spawn an angry bee, and it could also be useful for deleting/killing angry bees. 
The anger level Im using here (400-800) is the same that Mojang uses directly in NMS.

If you think I should remove it, I can remove it.

Reply:
as long is it uses whatever the default is i think it's nice